docs/herramientas/respaldos.md

```diff
@@ -4,8 +4,13 @@ title: Respaldos y Snapshots
 sidebar_label: Respaldos y Snapshots
 ---
 
-El servicio de snapshots de LACChain EOSIO le permitirá hacer cambios en los nodos de EOSIO rápidamente sin necesidad de sincronizar la cadena de bloques desde su génesis.
+El servicio de snapshots y respaldos de LACChain EOSIO le permitirá desplegar y hacer cambios en los nodos de EOSIO rápidamente sin necesidad de sincronizar la cadena de bloques desde su génesis.
+
+En el siguiente enlace puede encontrar una explicacion acerca de como funcionan los diferentes respaldos y snapshots en EOSIO: [Nodeos Replays](https://developers.eos.io/manuals/eos/v2.0/nodeos/replays/index).
+
+Actualmente LACChain EOSIO provee de varios respaldos y snapshots que usted puede descargar del siguiente enlace: [http://archive.eosio.cr/](http://archive.eosio.cr/).
+Con dichos respaldos o snapshots usted podrá crear un nodo de LACChain de forma mucho más rápida, sin la necesidad de sincronizar desde el bloque génesis. 
 
 :::note Trabajo en progreso
-Aun estamos trabajando en este aspecto de la documentación, si desea mejorar el contenido puede aprender [como contribuir](../guias/contribuir). Consulte el [Roadmap del proyecto](../roadmap).
-:::
\ No newline at end of file
+Aun estamos trabajando en este aspecto de la documentación, si desea mejorar el contenido puede aprender [cómo contribuir](../guias/contribuir). Consulte el [Roadmap del proyecto](../roadmap).
```

Comment:
@AngeloCG97 

seria bueno hacer un find and replace de `[cómo contribuir](../guias/contribuir).`

Reply:
De acuerdo voy a realizar el issue